RELEASE-NOTES.md

```diff
@@ -1,5 +1,25 @@
 # Prefect Release Notes
 
+## 2.0.4
+
+### Simplified deployments
+The deployemnts experience has been refined to remove extraneous artifacts and make configuration even easier. In particular:
+
+-   `prefect deployment build` no longer generates a  `manifest.json` file. Instead, all of the relevant information is written to the `deployment.yaml` file.
+- Values in the `deployment.yaml` file are more atomic and explicit
+-   Local file system blocks are no longer saved automatically
+-   Infrastructure block values can now be overwritten with the new `infra_overrides` field
+
+### Start custom flow runs from the UI
+Now, from the deployment page, in addition to triggering an immediate flow run with default parameter arguments, you can also create a custom run. A custom run enables you to configure the run's parameter arguments, start time, name, and more, all while while otherwise using the same deployment configuration. The deployment itself will be unchanged and continue generate runs on its regular schedule.
```

Comment:
```suggestion
Now, from the deployment page, in addition to triggering an immediate flow run with default parameter arguments, you can also create a custom run. A custom run enables you to configure the run's parameter arguments, start time, name, and more, all while otherwise using the same deployment configuration. The deployment itself will be unchanged and continue to generate runs on its regular schedule.
```